custom/icds_reports/templates/icds_reports/icds_app/progress-report.directive.html

```diff
@@ -72,6 +72,9 @@ <h2>No Data</h2>
             <div class="alert alert-info" ng-show="$ctrl.showWarning">
                 <strong>Info!</strong> "You are viewing data for the current calendar month. Data for this report is calculated using calendar months (for example, January 1st-January 31st). Data for this month will not be completed until the end of the month."
             </div>
+            <div class="alert alert-info" ng-show="$ctrl.showPreviousMonthWarning">
```

Comment:
I don't know much about angular so this may be an obvious question.

Why does `$ctrl.showWarning()` in the download.directive file need `()` but this doesn't?

Reply:
As here it is function and there it is variable that is recalculated on filters change